File: tests/hurst_channel_test.py
import numpy as np
from taflow import HurstChannel
def test_hurst_channel_lifecycle():
    s=HurstChannel(np.array([],float),np.array([],float),np.array([],float),2,.5);s.extend([2,3],[1,2],[1.5,2.5]);assert s.value is not None;s.reset();assert len(s)==0

File: crates/taflow-core/src/indicators/hurst_channel_test.rs
use super::hurst_channel::HurstChannel;
#[test]
fn lifecycle() {
    let mut s = HurstChannel::new(2, 0.5).unwrap();
    s.append(2.0, 1.0, 1.5);
    assert!(s.append(3.0, 2.0, 2.5).is_some());
    s.reset();
    assert!(s.value().is_none());
}
